docs/guide_publish_cogs.rst

```diff
@@ -86,3 +86,12 @@ Keys specific to the cog info.json (case sensitive)
 
 .. warning::
     Shared libraries are deprecated since version 3.2 and are marked for removal in the future.
+
+Adding to the Index
+-------------------
+
+Repositories that are correctly configured can be added to the `public index of cogs <https://index.discord.red/>`_.
+
+To be added to the index, make a pull request to the `Red-Index repository <https://github.com/Cog-Creators/Red-Index>`_ in the unnapproved section.
```

Comment:
Just "unapproved section" looks a bit confusing at first, so tried to address that. Also fixed a small typo.
```suggestion
To be added to the index, make a pull request to the `Red-Index repository <https://github.com/Cog-Creators/Red-Index>`_ in the unapproved section. You can learn more about this process in the repository description.
```